Wrappers/Python/cil/framework/acquisition_geometry.py

```diff
@@ -1798,7 +2045,7 @@ def get_centre_of_rotation(self, distance_units='default', angle_units='radian')
         if hasattr(self.config.system, 'calculate_centre_of_rotation'):
             offset_distance, angle_rad = self.config.system.calculate_centre_of_rotation()
         else:
-            raise NotImplementedError
+            raise NotImplementedError()
```

Comment:
I think this is a `TypeError` as above

Reply:
https://github.com/TomographicImaging/CIL/blob/bbbe088ed4006ae36392ad7dd9b108d7523d2725/Wrappers/Python/cil/framework/acquisition_geometry.py#L2034-L2037